### PR TITLE
AddOn updated for projectPath macor and iconPath entry

### PR DESCRIPTION
**Pull request by ifekonja**

### CODE REVIEW

File: project/elements-custom.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (jogibear9988) [inline project/elements-custom.json:1]: this is a file, special for your solution, it should not be in this repo

**Comment by jogibear9988** [comment]
also these images, have nothing to do with standard html components, so they also should not be in this repo.

**Comment by jogibear9988** [comment]
Maybe we need to add a way, how the extension load images from the project you are openeing, or a setting where you can pass a path for additional elements

**Comment by jogibear9988** [comment]
And how do your elements work? do you load the JS form them in the HTML you edit?

**Comment by ifekonja** [comment]
1) you are right; I have to think on plain html browser (no macro)
2) same valids for images, template in code is enough to cathc the idea
3) I am thinking about the way to get the system "open" for "project oriented" solution, without core changes

**Pull request closed without merging**